src/js/organisms/filter-form.js

```diff
@@ -74,24 +107,31 @@ module.exports = class FilterForm {
    *
    * @param {Event} e
    */
-  uncheckInputs(e) {
+  clearInputs(e) {
     e.preventDefault();
-    const { controls } = e.target.dataset;
-    const checkboxes = this.form.querySelectorAll('input[type=checkbox]');
-    const matchedCheckboxes = Array.from(checkboxes).filter(
-      (checkbox) =>
-        (!controls || checkbox.name === controls) && checkbox.checked,
+    const triggerEl = e.target.closest('a.js-filter-clear');
+    const { controls } = triggerEl.dataset;
+    const formInputs = this.form.querySelectorAll(
+      'input[type=checkbox], input[name="search"]',
```

Comment:
If we have the search input be a `type="search"` then we can change that here and not be concerned about the value of `name` changing from under us ;)